webapp/src/ts/services/enketo.service.ts

```diff
@@ -143,11 +90,6 @@ export class EnketoService {
       ])
       .then(([html, model]) => {
         const $html = $(html);
-        $html.find('[data-i18n]').each((idx, element) => {
-          const $element = $(element);
-          $element.text(this.translateService.instant('enketo.' + $element.attr('data-i18n')));
-        });
```

Comment:
Extracted to the `EnketoFormService.replaceDataI18nTranslations` method.  The rest of this logic needs to stay here, though, because we have to check the model to see if we need to actual build the contact-summary or not (before calling `EnketoFormService.renderForm`).